### DISCUSSION TITLE
posts/2022/05/tips-using-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#17] [Announcements]
# posts/2022/05/tips-using-giscus/

Some advice about a commenting system for static websites.

https://www.brycewray.com/posts/2022/05/tips-using-giscus/

**Comment by ThePlasmak** [comment]
Thank you for writing this—it helped me a lot when I was setting up giscus on my own website!

**Reply by brycewray** [reply]
@ThePlasmak glad to be of service!

**Comment by odhyp** [comment]
Thanks for the tips! :beer:

**Reply by brycewray** [reply]
@odhyp My pleasure!

**Comment by jumbosushi** [comment]
I was naively about to host the comments on my blog's repo discussions, but glad I saw this post 😅 I separated them into a new repo like recommended. Thanks for sharing it!

**Reply by brycewray** [reply]
@jumbosushi Yeah, definitely easier that way.
